### DISCUSSION TITLE
Create options picker

### DISCUSSION THREAD

**Discussion by EduZeroum** [Budibase/budibase#2004] [Help]
Hello i recently merged from budibase db to mysql , but in budibase db you could create a options picker from table but how can i do that with mysql table to appear options in budibase when i'm going to create a new row , please help 😢.

**Comment by mike12345567** [comment]
Hi @EduZeroum this has been discussed a fair bit, we plan to implement this soon, you can follow progress towards this as part of issue #1878.

**Reply by EduZeroum** [reply]
Hello, thanks for the brief reply. And i hope that this "idea" will be implemented soon.

**Reply by KernOil** [reply]
HI @EduZeroum  @mike12345567  the workaround to this problem is to create a form (BB schema)  inside that form have your selector. then create a sub-form (MySQL schema)  whith in the first form. this sub-form with all of the other fields you want to submit. then use the button to bring in those fields from the option picker. see below  
![image](https://user-images.githubusercontent.com/80058238/125973589-8955ebeb-9b43-4f70-a27b-01f6963d7e36.png)

now use the button to bring in the top drop-down menus

![image](https://user-images.githubusercontent.com/80058238/125973765-76fb812d-9403-4883-ad74-19382359a1d5.png)

this works great the problem is it jacks up your page formating a bit. you can use the custom CSS to eliminate the padding in the second form but its not perfect

**Reply by KernOil** [reply]
to fix the formating throw them all into a 'form group' will not be perfect but will smooth them out a bit
